internal/guest/runtime/hcsv2/workload_container.go

```diff
@@ -8,24 +8,25 @@ import (
 	"path/filepath"
 	"strings"
 
-	"github.com/Microsoft/hcsshim/internal/oc"
-	"github.com/Microsoft/hcsshim/pkg/annotations"
 	oci "github.com/opencontainers/runtime-spec/specs-go"
 	"github.com/pkg/errors"
 	"go.opencensus.io/trace"
 	"golang.org/x/sys/unix"
+
+	"github.com/Microsoft/hcsshim/internal/constants"
+	"github.com/Microsoft/hcsshim/internal/oc"
+	"github.com/Microsoft/hcsshim/pkg/annotations"
 )
 
 func getWorkloadRootDir(id string) string {
-	return filepath.Join("/run/gcs/c", id)
+	return filepath.Join(constants.LCOWRootPrefixInUVM, id)
 }
 
 func updateSandboxMounts(sbid string, spec *oci.Spec) error {
-	sandboxMountPrefix := "sandbox://"
 	for i, m := range spec.Mounts {
-		if strings.HasPrefix(m.Source, sandboxMountPrefix) {
+		if strings.HasPrefix(m.Source, constants.SandboxMountPrefix) {
```

Comment:
should we also move the `hugepages://` string out below?

Reply:
yeah, I should've as well.